_posts/2021-02-15-changelog-141.markdown

```diff
@@ -0,0 +1,96 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #141"
+date: 2021-02-10 17:00:00 +0300
+---
+
+
+## New Features
```

Comment:
Let's add additional item about debugging support in GoLand 2021.1.
There isn't teh corresponding PR since all chages were made in GoLand.
See https://github.com/intellij-rust/intellij-rust/issues/5422#issuecomment-778158181 for more details.